CHANGES.md

```diff
@@ -102,7 +102,18 @@ To be released.
     already has and deal with only longer (higher) blocks.
  -  Fixed a bug that occured when `Swarm` was handling multiple responses at the
     same time.
+ -  Removed following methods in `IStore` and `AddressTransactionSet` that
```

Comment:
The `AddressTransactionSet` class in itself was also removed (not only those methods in it).

Reply:
it means 'methods in `IStore`' and '`AddressTransactionSet`'. should I re-write like "methods in `IStore` and `AddressTransactionSet` class"?

Reply:
Ah, make sense.  As the statement is ambiguous, we would disambiguate it by changing the word order like:

> Removed `AddressTransactionSet` which handles handle `Address` to `IEnumerable<TxId>` indices, and the following methods in `IStore`.